packages/iTwinUI-react/src/core/ComboBox/ComboBox.tsx

```diff
@@ -223,17 +241,19 @@ export const ComboBox = <T,>(props: ComboBoxProps<T>) => {
   }, [isOpen]);
 
   // Update filtered options to the latest value options according to input value
-  const [filteredOptions, setFilteredOptions] = React.useState(options);
+  const [filteredOptions, setFilteredOptions] = React.useState(
```

Comment:
This change is not necessary as `useState` uses value passed only for initialization later it is not used.

Reply:
undo